crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -291,6 +300,101 @@ where
     }
 }
 
+/// A convenient abstraction for adding all the systems necessary for a binned
+/// render phase to the render app.
+///
+/// This is the version used when the pipeline supports GPU preprocessing: e.g.
+/// 3D PBR meshes.
+pub struct BinnedRenderPhasePlugin<BPI, GFBD>(PhantomData<(BPI, GFBD)>)
```

Comment:
These plugins should probably be mentioned in the migration guide section of the PR description. Just something simple like "If you were using `batch_and_prepare_render_phase` you should use `BinnedRenderPhasePlugin` or `SortedRenderPhasePlugin`"

Reply:
OK, I did that.